src/AddTodoForm.js

```diff
@@ -1,21 +1,27 @@
 import React from 'react';
 
-function AddTodoForm (props) {
+function AddTodoForm ({onAddTodo}) {
 
-    function handleAddTodo(event) {
-        event.preventDefault();
-        const todoTitle = event.target.title.value;
+    const [todoTitle, setTodoTitle] = React.useState('');
+
+    function handleTitleChange (e) {
+        const newTodoTitle = e.target.value;
+        setTodoTitle(newTodoTitle);
+    }
+
+    function handleAddTodo(e) {
+        e.preventDefault();
         console.log(todoTitle);
-        event.target.reset();
-        props.onAddTodo(todoTitle);
+        onAddTodo({ title:todoTitle, id:Date.now() });
+        setTodoTitle('');
     }
     
     return (
         <form onSubmit={handleAddTodo}>
             <label htmlFor='todoTitle'>Title</label>
-            <input id="todoTitle" name="title"/>
+            <input id="todoTitle" name="todoTitle" value={todoTitle} onChange={handleTitleChange} />
```

Comment:
Everything looks good but I'd recommend adding a `required` attribute so that empty string todos are considered invalid.